DataConnectors/AIVectraDetect.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "AIVectraDetect",
+    "title": "AI Vectra Detect",
+    "publisher": "Vectra AI",
+    "descriptionMarkdown": "The AI Vectra Detect connector allows users to connect Vectra Detect logs with Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. This gives users more insight into their organization's network and improves their security operation capabilities.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "AI Vectra Detect X Series",
+            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Vectra Networks\"\n| where DeviceProduct == \"X Series\"\n"
+        }
+    ],
+    "sampleQueries": [
```

Comment:
The sample queries are very lean. Please include at least two additional queries with say, certain types of activities or other interesting columns that might be interesting from a AI Vectra customer standpoint.

Reply:
I added more sample queries in the json file.